aws/provider.go

```diff
@@ -832,6 +832,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_default_subnet":                                      resourceAwsDefaultSubnet(),
 			"aws_subnet":                                              resourceAwsSubnet(),
 			"aws_swf_domain":                                          resourceAwsSwfDomain(),
+			"aws_synthetics_canary":                                   resourceAwsSyntheticsCanary(),
```

Comment:
Should fix up the CI error then make this good to go! Thanks for splitting this off. 😄 

```suggestion
```